integration_tests/dbt_project.yml

```diff
@@ -43,6 +43,7 @@ vars:
     by_database: ['dummy'] # dummy inputs
     by_schema: ['dq_tools_mart'] # exclude all dq_tools.03_mart
     by_table: ['dq_issue_log', 'dummy'] # dummy inputs
+  dq_tools__auto_generate_test_description: "1"
```

Comment:
Is it better to keep this consistent with the `dq_tools_enable_store_test_results` variable by using `True / False` instead?

Reply:
Yep makes sense, but using `var("dq_tools__auto_generate_test_description", "0") == "1"` ensures a clear, unambiguous evaluation in Jinja, especially when dealing with variables sourced from environments or configurations

Reply:
@jingyu-infinitelambda We might need to refactor this bit later on since the naming is quite messy now. Therefore, this should be a blocker to merge PR now.